internal/provider/utils_test.go

```diff
@@ -43,6 +46,15 @@ func checkBasicJobAttributes(t *testing.T, name string, statusPattern *regexp.Re
 	)
 }
 
+//nolint:unparam // keeping name parameter for future test reuse
```

Comment:
The `name` parameter appears to be used. Is the `nolint` directive still needed?

Reply:
That makes sense. Thank you for explaining that.